### ISSUE TITLE
mac app crashing

### ISSUE THREAD

**Issue by VikramTiwari** [metabase/metabase#6068]
- Your operating system: OS X 10
- Metabase version: 0.26.1

Upgrade using the normal update in mac app and it results in breaking app where it doesn't launch anymore.

![screen shot 2017-09-28 at 13 53 21](https://user-images.githubusercontent.com/1330677/30990154-0a628ea4-a455-11e7-8a80-1beb71c79c23.png)
![screen shot 2017-09-28 at 13 55 12](https://user-images.githubusercontent.com/1330677/30990156-0ba8227e-a455-11e7-8a03-3ac0976356b8.png)

**Comment by camsaul** [comment]
Hi @VikramTiwari, does quitting and restarting the app fix it for you?

**Comment by VikramTiwari** [comment]
Nope. Tried it a bunch.

**Comment by dennisberlin** [comment]
This is happening to me as well. Also restarting the entire system did not help. I'm on macOS 10.12.6.

Any help in getting it back up is appreciated!

**Comment by camsaul** [comment]
@VikramTiwari @dennisberlin I don't know exactly what's wrong but I did a new build of the Mac App which I'm hoping will fix your issues. Do me a favor and download it from [downloads.metabase.com/v0.26.1/Metabase.dmg](http://downloads.metabase.com/v0.26.1/Metabase.dmg). Then replace the one you were trying earlier with that (make sure you close it first). Let me know if that fixes things for you

**Comment by VikramTiwari** [comment]
Did not work. I am gonna try cleaning up all the temp files etc to see if it's something in there.

**Comment by dennisberlin** [comment]
@camsaul for me the new build worked fine, thanks! (and sorry for late reply, we had a public holiday in Germany).

**Comment by camsaul** [comment]
@VikramTiwari I'm still struggling to reproduce this issue.

Can you try downgrading to 0.25.2 (http://downloads.metabase.com/v0.25.2/Metabase.dmg) and seeing if that launches?

**Comment by fandomfilms** [comment]
Downgrading resolved my issue

**Comment by VikramTiwari** [comment]
0.25.2 worked.
Upgraded to the latest and failed again. Seems like it's the same as #4246

![screen shot 2017-10-05 at 07 48 39](https://user-images.githubusercontent.com/1330677/31233521-b227d29c-a9a1-11e7-814c-73b314ec9db9.png)

**Comment by dennisberlin** [comment]
@camsaul unfortunately, the new build of 0.26.1 stopped working after a restart :( now back to 0.25.2 which works fine.

**Comment by camsaul** [comment]
Good news @dennisberlin @VikramTiwari @fandomfilms @dennisberlin. I think I got to the bottom of the issues with the Mac App. I uploaded a new build here:

http://downloads.metabase.com/v0.26.1/Metabase-0.26.1.1.dmg

if you guys could give it a try and let me know if launches without problems for you, that would be great. If it doesn't, I added some code to make the last few log messages show up on the "Fatal Error" alert box so that should make diagnosing problems a little easier.
